src/controllers/UserController.js

```diff
@@ -39,12 +39,21 @@ export default class UserController {
    * @param  {object} res
    * @return {object} return all users in database
    */
-  static async getAllUser(req, res) {
-    const role = 'normal';
-    const findAll = await User.getAllUser({ role });
+  static async getAll(req, res) {
+    const role = req.user.role === 'normal' ? 'normal' : null;
```

Comment:
I don't think we should have the role check here. we have to keep them in a helper file